moonbeam-types-bundle/README.md

```diff
@@ -3,20 +3,74 @@
 Exports npm package `moonbeam-types-bundle`, formated as per polkadot-js specification to use
 with the app or the api.
 
-## Exports
+## ⚠️Warning: Types deprecation⚠️
 
+Following runtime upgrade 900 (include substrate v0.9.11), types are now retrieved from the node, in
+a **camelCase** format.
+
+A **new version** has been released `moonbeam-types-bundle@2.0.0`.  
```

Comment:
... for people who want to keep on using pre 900 runtime and get the Camel Cased types.